pointsource_imaging/MIRI_Aperture_Photometry_solution.ipynb

```diff
@@ -161,14 +161,27 @@
     "### 2.2<font color='white'>-</font>Plot configuration<a class=\"anchor\" id=\"plot\"></a> ###"
```

Comment:
<p>With the blank lines of code now filled, I would say that Section 8 and its entry in the table of contents are no longer needed in this version of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='134'/>

Reply:
I removed these parts from the solution version and left them only in the live version.

Reply:
The removal of the cells did not make it through to the commit, but I went ahead removed them since your intended to do so.